package.json

```diff
@@ -50,43 +50,43 @@
   "dependencies": {
     "@terrestris/base-util": "^1.0.1",
     "@turf/turf": "^6.5.0",
-    "@types/geojson": "^7946.0.8",
-    "geostyler-openlayers-parser": "^3.1.0",
+    "@types/geojson": "^7946.0.10",
```

Comment:
kind of unrelated, but shouldn't this be moved to `devDependencies`?

Reply:
Definitely, moved.